RevenueCatUI/CustomerCenter/Data/Transaction.swift

```diff
@@ -56,4 +60,10 @@ extension NonSubscriptionTransaction: Transaction {
     var managementURL: URL? {
         nil
     }
+
+    var price: ProductPaidPrice? {
+        // We don't have that information in the CustomerInfo
+        nil
```

Comment:
`.unknown`?

Reply:
This is tricky. `RevenueCatUI.Transaction` has `var price: ProductPaidPrice? { get }`.

`CustomerInfo` provides `ProductPaidPrice` only for `SubscriptionInfo`, so we're adding it for `NonSubscriptionTransaction` to simplify the handling inside `PurchaseInformation`.

I think that eventually we will get rid of `RevenueCatUI.Transaction` because it's just another layer of abstraction that's used ONLY in the initializer.